my_game.py

```diff
@@ -121,8 +123,9 @@ def update(self):
         # Update y position
         self.center_y += self.change_y
 
-        # Remove shot when over top of screen
-        if self.bottom > SCREEN_HEIGHT:
+        # Has a range of how long the shot can last for.
+        self.shot_range += 1
```

Comment:
Please calculate how long the shot has actually traveled. rename _self.shot_range_ to _self.distance_traveled_.